src/F1_plotting.py

```diff
@@ -261,6 +261,40 @@ def draw_graph(
         )
 
 
+def get_color(idx_line, color_list=None):
+    """Pick a color within a color list with periodic boundary conditions
+
+    Parameters
+    ----------
+    idx_line: int
+        index of the line in a plot for which a color is required
+
+    colors: list of str or list to tuple (hexadecimal or rbg code)
+        list of colors
+        Default: None
+
+    Returns
+    -------
+    The color in the color list corresponding to the index modulo the color list length
```

Comment:
"modulo"?

Reply:
a modulo b  is the operation which returns the rest of the integer division of a by b